src/main/java/com/google/cloud/tools/libraries/libraries.json

```diff
@@ -19,8 +19,56 @@
          "groupId" : "com.google.cloud",
          "artifactId" : "google-cloud-bigquery",
          "version" : "0.22.0-beta"
+       }
       }
-    }
-   ]
+    ]
+  },
+  {
+    "name": "Google Cloud Datastore",
+    "documentation": "https://cloud.google.com/datastore/docs/reference/libraries",
+    "description":"Google Cloud Datastore is a NoSQL key-value database built for automatic scaling.",
+    "transport": "http",
+    "icon":"https://cloud.google.com/_static/images/cloud/products/logos/svg/datastore.svg",
+    "clients": [
+      {
+      "name":"Cloud Datastore Client Library for Java",
+      "language":"java",
+      "site":"https://cloud.google.com/datastore/docs/reference/libraries#client-libraries-install-java",
+      "apireference":"https://googlecloudplatform.github.io/google-cloud-java/latest/apidocs/index.html?com/google/cloud/datastore/package-summary.html",
+      "infotip":"The com.google.cloud.datastore packages",
+      "status" : "GA",  
+      "source" : "https://github.com/GoogleCloudPlatform/google-cloud-java/tree/master/google-cloud-bigquery",
+      "languageLevel" : "1.7.0",
+      "mavenCoordinates" : {
+         "groupId" : "com.google.cloud",
+         "artifactId" : "google-cloud-datastore",
+         "version" : "1.4.0"
+       }
+      }
+    ]
+  },
+  {
+    "name": "Google Cloud Storage",
+    "documentation": "https://cloud.google.com/storage/docs/reference/libraries",
+    "description":"Unified BLOB storage for developers and enterprises, from live data serving to data analytics/ML to data archiving.",
+    "transport": "http",
+    "icon":"https://cloud.google.com/_static/images/cloud/products/logos/svg/storage.svg",
+    "clients": [
+      {
+      "name":"Google Cloud Storage Client Library for Java",
+      "language":"java",
+      "site":"https://cloud.google.com/storage/docs/reference/libraries#client-libraries-install-java",
```

Comment:
Wouldn't the `#using_the_client_library` section be more appropriate, since the user will be using the consuming UI to automate the installation of this library?

Reply:
What I really wanted was the entire page, but that won't pick up Java as opposed to Python. This is the nearest link to the top.

Reply:
Let me file a bug requesting a URL for the whole page.